style.css

```diff
@@ -297,3 +301,116 @@
   color: #7a869a;
   rotate: 90deg;
 }
+
+.contact-h {
+  font-family: " Poppins", sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+}
+
+.contact-p {
+  font-family: " Poppins", sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #ebebff;
+  width: 100%;
+}
+
+.contact-section {
+  background-color: #6070ff;
+  background-image: url(image-folder/Contact1.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 779px;
+  width: 315px;
+  border-top-left-radius: 100px;
+  padding: 8px;
+  display: flex;
+  flex-direction: column;
+  margin-left: 330px;
+}
+
+.contact-section h3 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  margin-left: 40px;
+  margin-right: 20px;
+  margin-bottom: 20px;
+  margin-top: 120px;
+}
+
+.form-p {
+  display: flex;
+  margin-left: 12px;
+  margin-right: 20px;
+  margin-bottom: 300px;
+  flex-direction: column;
+}
+
+.input-p {
+  width: 80%;
+  height: 30px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+  padding: 15px 38px 15px 16px;
+}
+
+.input-t {
+  display: flex;
+  flex-direction: row;
+  width: 80%;
+  height: 150px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #172b4d;
+  padding: 15px 38px 15px 16px;
+}
+
+.btn-1 {
+  font-family: "Poppins", sans-serif;
+  align-items: center;
+  padding: 12px 16px;
+  width: 142px;
+  height: 48px;
+  background: #fff;
+  border-radius: 8px;
+  cursor: pointer;
+  color: #6070ff;
+}
```

Comment:
- [ ] Kindly make sure the border of the `Get in touch` button matches the provided Figma design.
- [ ] Kindly make sure to change the typography on the `Get in touch` button in order to match the provided Figma design. Kindly adjust your `font-weight` and `font-size`

     - ### YOUR DESIGN
     
     ![image](https://user-images.githubusercontent.com/87197412/230352735-4d0abeee-1080-4cdf-8556-21acba31ec97.png)

     - ### FIGMA DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/230352924-4b0580c8-3af2-44ff-b47b-77284396add5.png)